### PR TITLE
Add EvalAPI changes

### PR DESCRIPTION
**Pull request by zjesko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,41 +10,41 @@ git clone https://github.com/AIcrowd/aicrowd-example-evaluator
...
-  - `evaluation_pipeline_version` : A value ideally provided by the admins. For now, you can safely use the value `1.0`
-  - `authors` : Information about the authors of the evaluator
-  - `version` : version number of your evaluator
-
-* Update the `__init__.py` file with the relevant import file. Ideally it should follow the pattern `from .[evaluator_name] import [evaluator_class]`. In this example `from .example_evaluator import ExampleEvaluator`
-
-* Implement the actual evaluator class in `example_evaluator.py`. Do rename the file based on the value you chose for the `evaluator_name` field in `aicrowd.json`.
+* Update the [aicrowd.yaml](aicrowd.yaml) file with the following values
+  - `challenge.name` : This is the name of your challenge.
+  - `challenge.temlate` : This is the name of the template being used by us to create the evaluator. Do not change its value from `simple-evaluator`
```
Comment (skbly7) [inline README.md]: template

File: evaluator.py [hunk 2]
```diff
@@ -56,22 +55,28 @@ def _evaluate(self, client_payload, _context={}):
     _result_object["media_video_path"] = '/tmp/submission-video.mp4'
     _result_object["media_video_thumb_path"] = '/tmp/submission-video-small.mp4'
     """
+
+    assert "score" in _result_object
+    assert "score_secondary" in _result_object
+
     return _result_object
```
Comment (skbly7) [inline evaluator.py:62]: Feature request: Can we add example to add meta score field (maybe as different PR)?

File: README.md (hunk 1 above)
Comment (zjesko) [reply README.md]: fixed

**Review by skbly7 — APPROVED** [review]
tenor

**Comment by zjesko** [comment]
yeeeet

**Pull request merged**